java/src/main/java/dojo/supermarket/model/Discount.java

```diff
@@ -0,0 +1,26 @@
+package dojo.supermarket.model;
+
+public class Discount {
+
+    private final String description;
+    private final double discountAmount;
```

Comment:
while dealing with money it is recommended to use BigDecimal as it offers better precision 
```suggestion
    private final BigDecimal discountAmount;
```

Reply:
would be also nice to note that discountAmount should always have negative value (and add check in constructor method)